homeassistant/components/iotawatt/sensor.py

```diff
@@ -73,6 +83,7 @@ class IotaWattSensorEntityDescription(SensorEntityDescription):
         "WattHours",
         native_unit_of_measurement=ENERGY_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
+        icon="mdi:chart-histogram",
```

Comment:
```suggestion

```
```DEVICE_CLASS_ENERGY``` has a default icon type. I don't think it should be overwritten.

Reply:
i've removed it, I just don't like the default icon (same as power and power isn't energy)